src/main.cpp

```diff
@@ -12,6 +12,7 @@ int main( int argc, char* argv[] )
 
     LOG( INFO ) << settings::getSettingsAndValues();
 
+    QCoreApplication::setAttribute(Qt::AA_Use96Dpi);
```

Comment:
```suggestion
    QCoreApplication::setAttribute( Qt::AA_Use96Dpi );
```

this should fix up the clang formatting warnings (added spaces)